operator/pkg/controllers/etcd/pod.go

```diff
@@ -115,6 +116,19 @@ func podSpecFor(controlPlane *v1alpha1.ControlPlane) *v1.PodSpec {
 					},
 				},
 			}},
+			LivenessProbe: &v1.Probe{
+				Handler: v1.Handler{
+					HTTPGet: &v1.HTTPGetAction{
+						Host:   "127.0.0.1",
+						Scheme: v1.URISchemeHTTP,
+						Path:   "/health",
+						Port:   intstr.FromInt(2381),
+					},
+				},
+				InitialDelaySeconds: 120,
+				TimeoutSeconds:      15,
+				FailureThreshold:    8,
```

Comment:
Similarly, we can reduce TimeoutSeconds and FailureThreshold values so that kubelet can restart them quickly. Also, I checked the default `PeriodSeconds` is 10 seconds maybe we can reduce this as well to be 5 seconds?

Reply:
I reduced the InitialDelaySeconds, TimeoutSeconds, FailureThreshold, and Period for all liveness probes.